client/lib/cart-values/cart-items.js

```diff
@@ -283,6 +283,12 @@ function hasDomainRegistration( cart ) {
 	return some( getAll( cart ), isDomainRegistration );
 }
 
+function hasOnlyDomainRegistrationsWithPrivacySupport( cart ) {
+	return every( getDomainRegistrations( cart ), function( cartItem ) {
```

Comment:
Could probably be written as `every( getDomainRegistrations( cart ), privacyAvailable )`?